corehq/apps/accounting/static/accounting/js/subscriptions_main.js

```diff
@@ -5,6 +5,7 @@ import widgets from "accounting/js/widgets";
 import "accounting/js/base_subscriptions_main";
 
 $(function () {
+    debugger;
```

Comment:
😂 
And feel free to do a force push since this is the only change.

Reply:
hmm, I seem to have not committed the actual migration. Time for a new PR.